### PR TITLE
Coding Standards: Add visibility to methods in `tests/phpunit/tests/` A-B-C-D files.

### PR DESCRIPTION
**Pull request by costdev**
Coding Standards: Add visibility to methods in `tests/phpunit/tests/` A-B-C-D files.

"Visibility should always be declared"
See: https://make.wordpress.org/core/2020/03/20/updating-the-coding-standards-for-modern-php/

This commit applies the following sniffs to files in `tests/phpunit/tests/` A-B-C-D directories:
- Squiz.Scope.MethodScope
- PSR2.Methods.MethodDeclaration
- PSR2.Classes.PropertyDeclaration
- Squiz.WhiteSpace.ScopeKeywordSpacing

For most methods, these now indicate `public` visibility to avoid breaking backwards compatibility.
Any remaining methods now indicate `private` visibility where appropriate.

Additional fixups in this PR:
1. Methods touched by this PR have had unnecessary underscores removed from their names.

Trac ticket: https://core.trac.wordpress.org/ticket/54177

### CODE REVIEW

**Comment by hellofromtonya** [comment]
@costdev Can you rebase with `master` please to resolve the merge conflicts?

File: tests/phpunit/tests/ajax/ReplytoComment.php [hunk 1]
```diff
@@ -264,7 +264,7 @@ public function test_pre_comments_approved() {
 	/**
 	 * Blocks comments from being saved on 'pre_comment_approved', by returning WP_Error.
 	 */
-	function _pre_comment_approved_filter( $approved, $commentdata ) {
+	public function _pre_comment_approved_filter( $approved, $commentdata ) {
```
Comment (hellofromtonya) [inline tests/phpunit/tests/ajax/ReplytoComment.php:267]: @costdev This one is also a filter callback. To keep it consistent, I'd suggest:

```suggestion
	public function filter_pre_comment_approved_filter( $approved, $commentdata ) {
```

but also changing where it's registered to the filter.

File: tests/phpunit/tests/dependencies/wpScriptTag.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
  */
 class Tests_Functions_wpScriptTag extends WP_UnitTestCase {
 
-	function get_script_tag_type_set() {
+	public function get_script_tag_type_set() {
```
Comment (hellofromtonya) [inline tests/phpunit/tests/dependencies/wpScriptTag.php:11]: Hmm, this test needs a `test_` in front of it for it run. Outside the scope of this ticket. But it needs to be fixed.

Comment (costdev) [reply tests/phpunit/tests/dependencies/wpScriptTag.php:11]: 👍  Also needs:

1. A `$message` value for the two assertions or, ideally, a data provider.
2. A `@covers` tag for `::wp_get_script_tag`.
3. Some work on the test as both assertions currently fail for the following reasons:

Test #1 - `wp_get_script_tag()` returns the attributes in the order they were supplied. The expected value has the attributes in a different order than it supplied to `wp_get_script_tag()`.

Test #2 - `wp_get_script_tag()` calls `wp_sanitize_script_attributes()`. For boolean attributes, this returns `attribute="attribute"`. The test supplies `'nomodule' => true` and tests for `nomodule`, but gets `nomodule="nomodule"` instead. IMO, this should be part of a wider discussion in Core about the usage of `attribute="attribute"`. However, that is beyond the scope of fixing that test, which just needs some minor corrections in the assertions.

**Review by hellofromtonya — APPROVED** [review]
Well done @costdev. Ready for commit.

**Comment by hellofromtonya** [comment]
Committed via https://core.trac.wordpress.org/changeset/52010.

**Pull request closed without merging**